src/styles/blocks/row.scss

```diff
@@ -0,0 +1,24 @@
+.row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-basis: 100%;
+  box-sizing: border-box;
+
+  @media (min-width: 1024px) {
+    margin: 0 40px;
+  }
+
+  @media (min-width: 1200px) {
+    margin: 0 128px;
+  }
```

Comment:
why margins, but we had to use paddings

Reply:
yeah, missed that, while was refactoring structure
thanks a lot